mkdocs.yml

```diff
@@ -0,0 +1,29 @@
+theme:
+  name: material
+  palette:
+    primary: teal
+    accent: teal
+  icon:
+    repo: fontawesome/brands/github
+
+repo_url: https://github.com/BodenmillerGroup/ImcSegmentationPipeline
+repo_name: BodenmillerGroup/ImcSegmentationPipeline
+
+use_directory_urls: false
+
+site_name: IMC Segmentation Pipeline
+nav:
+    - Home: index.md
+    - Pre-processing: prepro.md
+    - Ilastik training: ilastik.md
+    - Cell segmentation: segmentation.md
+    - Cell measurement: measurement.md
+    - Output files: output.md
+    
+markdown_extensions:
+    - footnotes
```

Comment:
Are these extensions all shipped with barebones mkdocs-material? Otherwise I believe the docs workflow may need an update

Reply:
Yes, they are part of `mkdocs-material`